susemanager-utils/cluster-providers/caasp/formulas/caasp-management-node-formula/caasp-management-node-formula.spec

```diff
@@ -0,0 +1,54 @@
+#
+# spec file for package caasp-management-node-formula
```

Comment:
Shouldn't we have the formulas at the formulas repository?

Reply:
For now this is tightly coupled with the `susemanager-cluster-provider-caasp` package so I think it makes more sense to keep it here. Maybe in the future we'll move the `susemanager-cluster-provider-caasp.spec` package and its dependent formulas to a separate repo.